projects/plugins/jetpack/modules/widgets/search.php

```diff
@@ -421,11 +422,12 @@ public function widget_non_instant( $args, $instance ) {
 	 * @param array $instance The current widget instance.
 	 */
 	public function widget_instant( $args, $instance ) {
+		$blog_id = Jetpack::get_option( 'id' );
 		if ( Helper::should_rerun_search_in_customizer_preview() ) {
-			Jetpack_Search::instance()->update_search_results_aggregations();
+			Automattic\Jetpack\Search\Classic_Search::instance( $blog_id )->update_search_results_aggregations();
 		}
 
-		$filters = Jetpack_Search::instance()->get_filters();
+		$filters = Automattic\Jetpack\Search\Classic_Search::instance( $blog_id )->get_filters();
```

Comment:
These changes are not included in #21751.

Reply:
This comment is no longer applicable now that both Classic_Search and Instant_Search have been refactored to use `static:initialize` for initialization.